app/main.py

```diff
@@ -1,3 +1,6 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    count = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+    return count
```

Comment:
The task description recommends avoiding loops and using Python's built-in string functions. Consider using the `count()` method for a more concise solution.